### PR TITLE
[Kotahi DocMaps]: Fix review extraction bug

### PR DESCRIPTION
**Pull request by HazalCiplak**
https://github.com/elifesciences/data-hub-issues/issues/838

### CODE REVIEW

File: tests/unit_tests/kotahi_docmaps/v1/test_data.py [hunk 1]
```diff
@@ -88,14 +88,17 @@
     'manuscript_versions': [MANUSCRIPT_VERSION_1, MANUSCRIPT_VERSION_2]
 }
 
-REVIEW_1 = '''
-    Reviewer #1 (Public Review):
+SPACE_1 = ' '
```
Comment (de-code) [inline tests/unit_tests/kotahi_docmaps/v1/test_data.py]: I think `SPACE` probably doesn't need a numbered suffix. Or if we did use variations of whitespaces, then we would likely name them.

File: tests/unit_tests/kotahi_docmaps/v1/test_data.py [hunk 2]
```diff
@@ -88,14 +88,17 @@
...
+COLON_1 = ' '
```
Comment (de-code) [inline tests/unit_tests/kotahi_docmaps/v1/test_data.py]: It says colon, but is it? :-)

Do you think it is more readable with a colon constant?

File: tests/unit_tests/kotahi_docmaps/v1/test_data.py (hunk 1 above)
Comment (HazalCiplak) [reply tests/unit_tests/kotahi_docmaps/v1/test_data.py]: I can remove it. But I thought we are doing it even we don’t have many with same name. When we look at the naming in our tests they always has number suffix.

File: tests/unit_tests/kotahi_docmaps/v1/test_data.py (hunk 2 above)
Comment (HazalCiplak) [reply tests/unit_tests/kotahi_docmaps/v1/test_data.py]: ah yes :D

**Review by de-code — APPROVED** [review]

**Pull request merged**